Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.test_generator.services;
+
+import com.epam.test_generator.dao.interfaces.CaseDAO;
+import com.epam.test_generator.dao.interfaces.SuitDAO;
+import com.epam.test_generator.entities.Case;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+public class CaseService{
+
+    @Autowired
+    private CaseDAO caseDAO;
+
+    @Autowired
+    private SuitDAO suitDAO;
+
+    @Transactional
```

Comment:
You can use this annotation on the class level instead to make all methods transactional